app/shop.py

```diff
@@ -0,0 +1,41 @@
+import datetime

```

Comment:
The task checklist specifies to use `import datetime` exactly, which is correctly done here. No issues with this line.